.github/workflows/cross-version-tests.yml

```diff
@@ -60,8 +60,24 @@ jobs:
         run: |
           conda install python=3.6
           python --version
+      - name: Get cache key
+        env:
+          INSTALL_COMMAND: ${{ matrix.install }}
+        id: get-cache-key
+        run: |
+          date=$(/bin/date -u "+%Y%m%d")
+          hash=$(echo -n "$INSTALL_COMMAND" | sha256sum)
+          echo "::set-output name=key::$date-$hash"
```

Comment:
- Add a hash generated from `matrix.install` (which stores a string representing a commad to install required dependencies) because we should rebuild the wheel when `matrix.install` has changed.

Reply:
Also added a date so cache will expire in a day.